packages/building-rollup/README.md

```diff
@@ -268,11 +268,11 @@ export default deepmerge(basicConfig, {
 
 #### Resolve commonjs modules
 
-CommonJS is the module format for NodeJS, and not suitable for the browser. Rollup only handles es modules by default, but sometimes it's necessary to be able to import a dependency. To do this, you can add [rollup-plugin-commonjs](https://github.com/rollup/rollup-plugin-commonjs):
+CommonJS is the module format for NodeJS, and not suitable for the browser. Rollup only handles es modules by default, but sometimes it's necessary to be able to import a dependency if you want to use third party libraries in your web component. To do this, you can add [@rollup/plugin-commonjs](https://github.com/rollup/plugins/tree/master/packages/commonjs):
```

Comment:
I don't think adding this sentence is correct, not all third party libraries use commonjs.

Reply:
I explicitly added this line because I struggled to add a third party library however, you are correct. May be we can rewrite this as 'if you want to use third party libraries which only supports commonjs'.

Reply:
Yes that would be fine.

Reply:
Ok, then I will update the content

Reply:
I just updated with the discussed change.